src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -318,54 +302,34 @@ private static String formatInstant(Instant instant, ZoneId timeZone, String pat
         return zonedDateTime.format(formatter);
     }
 
-    @Deprecated
-    public static String formatDateTimeForSessions(Date dateInUtc, double sessionTimeZone) {
-        return formatDateTimeForSessions(
-                convertDateToInstant(dateInUtc), convertToZoneId(sessionTimeZone));
-    }
-
-    public static String formatDateTimeForSessions(Instant instant, ZoneId sessionTimeZone) {
-        return formatInstant(instant, sessionTimeZone, "EEE, dd MMM yyyy, hh:mm a 'UTC'Z");
-    }
-
     /**
-     * Formats a date in the format d MMM h:mm a. Example: 5 May 11:59 PM
+     * Formats an Instant for a session in a specified ZoneId.
+     *
+     * @param instant the instant to be formatted
+     * @param sessionTimeZone the ZoneId representing the timezone for that session
+     * @return a string in the format {@code EEE, dd MMM yyyy, hh:mm a 'UTC'Z}
      */
-    @Deprecated
-    public static String formatDateTimeForInstructorHomePage(Date date) {
-        return formatDateTimeForInstructorHomePage(convertDateToLocalDateTime(date));
+    public static String formatDateTimeForSessions(Instant instant, ZoneId sessionTimeZone) {
+        return formatInstant(instant, sessionTimeZone, "EEE, dd MMM yyyy, hh:mm a 'UTC'Z");
     }
 
     /**
      * Formats a date in the format d MMM h:mm a. Example: 5 May 11:59 PM
+     *
+     * @param localDateTime the java.time.LocalDateTime object to be formatted
```

Comment:
Standardise with `formatTime12H` and everywhere else how to refer to `LocalDateTime`.